sass/_header.scss

```diff
@@ -0,0 +1,33 @@
+.bg__color--blue {
+	.body__container--header {
+		background: no-repeat,
```

Comment:
Przecinek po _no-repeat_  jest zamierzony? Bo jeśli to nie ma być jakiś multiple background, to wszystkie właściwości powinny być oddzielone jedynie spacjami.

Reply:
dziękuję, przyjęto do świadomości :) i poprawiono